applications/StructuralMechanicsApplication/custom_processes/advance_in_time_strategy_high_cycle_fatigue_process.h

```diff
@@ -0,0 +1,112 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ \.
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics FemDem Application
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Sergio Jimenez Reyes
+//
+
+#if !defined(KRATOS_ADVANCE_IN_TIME_STRATEGY_HIGH_CYCLE_FATIGUE_PROCESS)
+#define KRATOS_ADVANCE_IN_TIME_STRATEGY_HIGH_CYCLE_FATIGUE_PROCESS
+
+#include "processes/process.h"
+#include "includes/model_part.h"
+#include "structural_mechanics_application_variables.h"
+
+namespace Kratos
+{
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+/**
+ * @class AdvanceInTimeStrategyHighCycleFatigueProcess
+ * @ingroup StructuralMechanicsApplication
+ * @brief This class determines the advance in time to be performed for a regular cyclic load for the high cycle fatigue CL
+ * @author Sergio Jimenez
+ */
+class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION)AdvanceInTimeStrategyHighCycleFatigueProcess : public Process
```

Comment:
```suggestion
class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION)AdvanceInTimeHighCycleFatigueProcess : public Process
```

Strategy is reserved for other classes